### PR TITLE
Fix cucushift test mapping from QE

### PR DESCRIPTION
**Pull request by liangxia**
/cc @stbenjam @jhou1

### CODE REVIEW

**Comment by stbenjam** [comment]
Some formatting issues in the go code, you can use an IDE to `go fmt` to fix it

File: pkg/components/imageregistry/component.go [hunk 1]
```diff
@@ -33,9 +33,15 @@ var ImageRegistryComponent = Component{
...
+			{
+				IncludeAny: []string{
+					":ImageRegistry ",
+				},
+				Priority: 1,
+			},
 			{Suite: "ImageStream Manifest"},
 			{Suite: "Image_Registry"},
-			{Suite: "remote registry related scenarios"},
+			// {Suite: "remote registry related scenarios"},
```
Comment (stbenjam) [inline pkg/components/imageregistry/component.go:44]: Why is this commented out? If it's unneeded it should just get removed

File: pkg/obsoletetests/ocp_obsoletetests.go [hunk 2]
```diff
@@ -20,6 +20,10 @@ var obsoleteTests = sets.New[obsoleteTestIdentifier](
 			name:  "[sig-arch] Check if alerts are firing during or after upgrade success",
 			suite: "Cluster upgrade",
 		},
+		{
+			name:  "OCP-12158:APIServer Specify ResourceQuota on project",
+			suite: "remote registry related scenarios",
+		},
```
Comment (stbenjam) [inline pkg/obsoletetests/ocp_obsoletetests.go:26]: What's the background for marking this obsolete?

File: pkg/components/imageregistry/component.go (hunk 1 above)
Comment (liangxia) [reply pkg/components/imageregistry/component.go]: Removed this line.

File: pkg/obsoletetests/ocp_obsoletetests.go (hunk 2 above)
Comment (liangxia) [reply pkg/obsoletetests/ocp_obsoletetests.go:28]: Add the context to obsolete this test case in the PR.

**Comment by liangxia** [comment]
@stbenjam Could you help take a look ?

**Comment by liangxia** [comment]
@stbenjam Could you help check this PR ?

**Pull request merged**